cookbooks/fb_collectd/README.md

```diff
@@ -0,0 +1,39 @@
+fb_collectd Cookbook
+====================
+The `fb_collectd` cookbook installs and configures collectd, a statistics
+collection and monitoring daemon.
+
+Requirements
+------------
+This cookbook requires CentOS, Debian or Ubuntu.
+
+Attributes
+----------
+* node['fb_collectd']['collection']
+* node['fb_collectd']['globals']
+* node['fb_collectd']['plugins']
+* node['fb_collectd']['sysconfig']
+
+Usage
+-----
+### Daemon
+To install collectd include `fb_collectd::default`. Global settings can be
+customized using the `node['fb_collectd']['globals']` attribute and individual
+plugins can be enabled and configured using `node['fb_collectd']['plugins']`.
+Example:
+
+  node.default['fb_collectd']['global']['FQDNLookup'] = true
```

Comment:
this won't render properly. You either need 4 spaces or you need triple-backticks. Prefer triple-backticks.